packages/astro-angular/src/client.ts

```diff
@@ -1,12 +1,12 @@
 import 'zone.js/dist/zone.js';
-import { createApplication } from '@angular/platform-browser';
+import type { ɵComponentType as ComponentType } from '@angular/core';
 import { ApplicationRef, NgZone, createComponent } from '@angular/core';
-import type { ɵComponentType } from '@angular/core';
+import { createApplication } from '@angular/platform-browser';
 
 export default (element: HTMLElement) => {
   return (
-    Component: ɵComponentType<unknown>,
-    _props?: unknown,
+    Component: ComponentType<unknown>,
+    _props?: Record<string, unknown>,
```

Comment:
I used the underscore for unused arguments.

```suggestion
    props?: Record<string, unknown>,
```